app/src/auth/useApi.js

```diff
@@ -10,7 +10,7 @@ const makeApi = (accessToken) => {
     getWatchlist: () => _get("/api/watchlist"),
     getPortfolios: () => _get("/api/portfolios"),
     getPortfolioStocks: (portfolioID) =>
-      _get(`api/portfolios/${portfolioID}/stocks`),
+      _get(`/api/portfolios/${portfolioID}/stocks`),
```

Comment:
@xiaozhong21 This is the key fix.

Reply:
Thank you George! Can't believe I overlooked this... 😢

Reply:
@xiaozhong21 Easy to do when lots of stuff is happening.